backend/api/simulation.py

```
@@ -30,8 +31,10 @@
 
 router = APIRouter(prefix="/api/simulation", tags=["Simulation"])
 
-# OpenAI configuration
-openai.api_key = os.getenv("OPENAI_API_KEY")
+# OpenAI configuration - validate API key at module import
+if not settings.openai_api_key or not settings.openai_api_key.strip():
+    raise ValueError("OpenAI API key not configured in settings")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)